services/scabbard/libscabbard/src/client/mod.rs

```diff
@@ -68,7 +114,15 @@ impl ScabbardClient {
         let body = batches.into_bytes()?;
 
         debug!("Submitting batches via {}", url);
-        let request = Client::new().post(url).body(body);
+        // Allowing unused_mut because request must be mutable if experimental feature
+        // client-jwt is enabled, if feature is removed unused_mut notation can be removed
+        #[allow(unused_mut)]
+        let mut request = Client::new().post(url).body(body);
+
+        #[cfg(feature = "client-jwt")]
+        {
+            request = request.header("Authorizaiton", &self.auth);
```

Comment:
s/"Authorizaiton"/"Authorization", also throughout this file

Reply:
Fixed